src/applications/mhv-secure-messaging/components/shared/serverErrors.jsx

```diff
@@ -0,0 +1,19 @@
+export const edgeCaseBackendError = error => {
+  return {
+    title: 'Edge Case Backend Error',
+    detail: 'Edge Case Backend Error',
+    response: JSON.stringify(error),
+  };
+};
```

Comment:
would this content `Edge Case Backend Error` be user facing in case of a 503 error?

Reply:
agree. I was just thinking if this would be the case - would this wording `'Edge Case Backend Error'` be veteran facing (displayed in alert banner). If so, do we need to have this content be approved. If not, I think it is irrelevant to even have this to be returned as a part of this function.

Reply:
The response is for debugging purposes only. The rest of the functions only care about the title and defail.